### ISSUE TITLE
Dolby Vision Outputs Wrong Colors

### ISSUE THREAD

**Issue by carycary246** [the-database/mpv-upscale-2x_animejanai#20]
MPV is able to properly decode dolby vision files with the following configuration lines but when using animejanai, it displays the wrong colors.
```
profile=gpu-hq
vo=gpu-next
gpu-api=d3d11
gpu-context=d3d11
``` 
![mpv_2023-06-30_15-30-23](https://github.com/the-database/mpv-upscale-2x_animejanai/assets/31323333/276effdc-d378-42bc-9eaf-ca2e1fdbadd4)
With Animejanai 
![mpv_2023-06-30_15-30-17](https://github.com/the-database/mpv-upscale-2x_animejanai/assets/31323333/65bf2c01-3e20-4b5f-b0f1-5912288b59d8)

**Comment by 422415** [comment]
Unfortunately, it is not possible to upscale HDR color ranges from what I understand. You would need to look for the best way to tonemap HDR to SDR in mpv as that should work.

**Comment by carycary246** [comment]
Thanks for the insight but I don't believe color range is the issue here as I tested on different HDR movies and it upscales using animejanai perfectly fine. I also tested playing these movies in SDR with tonemapping and there are no issues. Only this dolby vision file exhibits this issue with incorrect colors.

**Comment by hooke007** [comment]
known issue, currently dv is not supported with vs in mpv.

**Comment by 422415** [comment]
> Thanks for the insight but I don't believe color range is the issue here as I tested on different HDR movies and it upscales using animejanai perfectly fine. I also tested playing these movies in SDR with tonemapping and there are no issues. Only this dolby vision file exhibits this issue with incorrect colors.

Yeah sorry I misspoke. I meant Dolby Vision not HDR. I looked into this when I tried to upscale Cyberpunk last year and had the same issue.

**Comment by 422415** [comment]
DV is now supported in MPV
